misk/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -43,7 +43,9 @@ data class DataSourceConfig(
   val fixed_pool_size: Int = 10,
   val connection_timeout: Duration = Duration.ofSeconds(30),
   val connection_max_lifetime: Duration = Duration.ofMinutes(30),
-  val migrations_path: String
+  val migrations_path: String,
+  // Threshold beyond which clock skew is considered unacceptable and the host is reported unhealthy.
+  val clock_skew_threshold: Duration = Duration.ofMinutes(1)
```

Comment:
I'm not sure what a good default is.

Reply:
I don’t think this needs to be configurable. If we pick a reasonable enough default I think that should be good for everyone.

My preference for the default is to warn at > 10s and become unhealthy at > 30s. This shouldn’t happen in practice.

Reply:
Fixed.